library/std/src/io/stdio.rs

```diff
@@ -1017,6 +1018,145 @@ where
     }
 }
 
+mod terminal_sealed {
+    #[unstable(feature = "sealed", issue = "none")]
+    pub trait Sealed {}
+}
```

Comment:
Is there a specific need to seal the `IsTerminal` trait? This prevents crates like cap-std and fs-err from implementing this for their own `File` types.

Reply:
Future-proofing, in case we want to introduce blanket implementations for a trait in the future.